integration-tests/src/test/resources/apps/buildDeployCoherenceAppInPod.sh

```diff
@@ -25,7 +25,7 @@ PORT=$2
 USER=${3:-weblogic}
 PASSWORD=${4:-welcome1}
 APP_DIR_INPOD=$5
-//APP_NAME=$6 - ignore this for coherence since the dir name and the app name are different.
+#APP_NAME=$6 - ignore this for coherence since the dir name and the app name are different.
 EAR_DEPLOY_TARGET=$7
```

Comment:
ok, but do you need to rename the others by one number less?? or are you just ignoring that value altogether, not removing it from the call?

Reply:
The script is being called from a utility method. Other tests are also using the same method to call their shell scripts to build and deploy. This argument is not needed for my script, so I just ignore it.